src/main/java/com/wishlist/exceptions/ProductNotFoundException.java

```diff
@@ -0,0 +1,9 @@
+package com.wishlist.exceptions;
+
+public class ProductNotFoundException extends RuntimeException {
+
```

Comment:
The message does not need to be stored as a field. We just want to pass it to the super class.